dev/WHAT_GOES_INTO_THE_NEXT_RELEASE.md

```diff
@@ -0,0 +1,131 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Do we have a process defined here?](#do-we-have-a-process-defined-here)
+- [How does the selection process for cherry-picking work?](#how-does-the-selection-process-for-cherry-picking-work)
+- [What's the release manager's role ?](#whats-the-release-managers-role-)
+- [Is this process following the ASF rules?](#is-this-process-following-the-asf-rules)
+- [What's the role of individual maintainers?](#whats-the-role-of-individual-maintainers)
+- [When proposed PRs are rejected?](#when-proposed-prs-are-rejected)
+- [Why are provider changes not cherry-picked?](#why-are-provider-changes-not-cherry-picked)
+- [What's the purpose of patch-level releases?](#whats-the-purpose-of-patch-level-releases)
+- [Do we intentionally skip over some changes when releasing a patchlevel release?](#do-we-intentionally-skip-over-some-changes-when-releasing-a-patchlevel-release)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+This page describes the context and the process we follow when we create a patchlevel release.
+
+# Do we have a process defined here?
+
+We have a well defined and well tested process that we follow.,
+We follow it since the beginning of Airflow 2.0 actually - it's been also done the same in 1.10 -
+but with some variations, we do it the same way since the beginning of 2.0,
+but it has been refined and improved over time - by those who volunteered their time in the release
+process (a lot of ad-hoc discussion have been traditionally happening in #release-management slack channel)
+and we continue to do so.
+
+The succinct form of it is described in a prominent place in our most important
+[README](../README.md#what-goes-into-the-next-release).
```

Comment:
I think this needs o be changed?

Reply:
I was thinking we are removing the one on README and having this one replace it. Obviously that's not the case, resolving this